pyriemann_qiskit/utils/hyper_params_factory.py

```diff
@@ -1,14 +1,98 @@
-from qiskit.circuit.library import ZZFeatureMap
+from qiskit.circuit.library import ZZFeatureMap, ZFeatureMap, PauliFeatureMap
 from qiskit.algorithms.optimizers import SPSA
 from qiskit.circuit.library import TwoLocal
 import inspect
 
 
+def gen_x_feature_map(reps=2):
+    """Return a callable that generates a XFeatureMap.
+
+    A feature map encodes data into a quantum state.
+    A XFeatureMap is a first-order Pauli-X evolution circuit (no entanglement).
+    See [1]_ for more details.
+
+    Parameters
+    ----------
+    reps : int (default 2)
+        The number of repeated circuits, greater or equal to 1.
+
+    Returns
+    -------
+    ret : PauliFeatureMap
+        An instance of PauliFeatureMap.
```

Comment:
We should be coherent with function name.
```suggestion
    ret : XFeatureMap
        An instance of XFeatureMap.
```